frontend/src/components/Election/Voting/VotePage.tsx

```diff
@@ -21,6 +21,8 @@ const CHECKED_BOX = "M 19 3 H 5 c -1.11 0 -2 0.9 -2 2 v 14 c 0 1.1 0.89 2 2 2 h
 //const UNCHECKED_BOX = "M 19 5 v 14 H 5 V 5 h 14 m 0 -2 H 5 c -1.1 0 -2 0.9 -2 2 v 14 c 0 1.1 0.9 2 2 2 h 14 c 1.1 0 2 -0.9 2 -2 V 5 c 0 -1.1 -0.9 -2 -2 -2 Z"
 const DOT_ICON = "M12 6c3.31 0 6 2.69 6 6s-2.69 6-6 6-6-2.69-6-6 2.69-6 6-6m0-2c-4.42 0-8 3.58-8 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8z"
 
+export const BallotContext = createContext({})
```

Comment:
Could you add types? Makes it easier to determine types in the components that use it.

Reply:
Sure, how deep should I show the nesting? Is it ok to only give the first level of keys, or do those need to be fully defined?

Reply:
Just the top level keys, should be able to import the race and candidate types so should be simple. 

What type do the components currently see this as? An any type?

Reply:
Done! (we figured out the format over slack)